.gitmodules

```diff
@@ -1,3 +1 @@
-[submodule "Scotch"]
-	path = Scotch
-	url = https://github.com/EasyPost/scotch.git
+
```

Comment:
Can this file be removed?

Reply:
You're right, I guess it could.

Reply:
Yes, this needs to be removed. You can follow this guide: https://gist.github.com/myusuf3/7f645819ded92bda6677